crates/sui-open-rpc/src/generate_json_rpc_spec.rs

```diff
@@ -84,7 +64,10 @@ async fn main() {
         Action::Print => {
             let content = serde_json::to_string_pretty(&open_rpc).unwrap();
             println!("{content}");
-            let (objects, txs, addresses) = create_response_sample().await.unwrap();
+            let network = start_rpc_test_network(Some(GenesisConfig::custom_genesis(1, 4, 30)))
```

Comment:
nit: can we define constants for `4`, `30` to avoid duplication and improve readability on line 79?